services/dynamic-sidecar/Dockerfile

```diff
@@ -52,6 +52,15 @@ RUN \
   --mount=type=bind,source=scripts/install_rclone.bash,target=install_rclone.bash \
   ./install_rclone.bash
 
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
```

Comment:
does this install a pinned version or always the latest?
I would tend to think that this is ok even with the latest.

Reply:
I would like the version to be pinned. I don't want unexpected behaviours between builds. Please pin the versions.

Reply:
Especially since these can happen between releasing to staging and then releasing to production will bring a broken build. Let's avoid this possibility

Reply:
pinned version :+1: